airbyte-webapp/src/views/Connector/ServiceForm/components/Controls/ConnectorServiceTypeControl.tsx

```diff
@@ -54,6 +88,31 @@ const ConnectorList: React.FC<MenuWithRequestButtonProps> = ({
   </>
 );
 
+const Option: React.FC<OptionProps> = (props) => {
+  return (
+    <components.Option {...props}>
+      <OptionView
+        data-testid={props.data.label}
+        isSelected={props.isSelected}
+        isDisabled={props.isDisabled}
+      >
+        <Text>
+          {props.data.img || null}
+          <Label>{props.label}</Label>
+        </Text>
+        {props.data.releaseStage ? (
```

Comment:
We should not show a stage label for GA connectors, they should simply go without a label.

Reply:
Fixed